### PR TITLE
austin.intro.2.puckett.james

### PR DESCRIPTION
**Pull request by JamesPuckett**

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+    <img src="https://c1552172.ssl.cf0.rackcdn.com/486743_x800.jpg"/>
+    <h4>Suspension</h4>
+    <p>Lorem ipsum dolor sit amet, noster quaeque an est, et cum autem recusabo efficiendi, ridens cetero democritum in mel. Legere aperiam suavitate vel id. Invenire partiendo cum id, quas aeque audire sea at. Mel tale lobortis moderatius cu, cu mei audiam iuvaret sententiae, te ubique nostrum nominavi vix. Iusto paulo nonumy vim cu, eu summo urbanitas philosophia mel.</p>
+    <img src="https://image.adam.automotive.com/f/45447471/mmfp-1302-01+kenny-brown-performance-suspension-install+kit"/>
+    <h4>Tires</h4>
+    <p>Lorem ipsum dolor sit amet, noster quaeque an est, et cum autem recusabo efficiendi, ridens cetero democritum in mel. Legere aperiam suavitate vel id. Invenire partiendo cum id, quas aeque audire sea at. Mel tale lobortis moderatius cu, cu mei audiam iuvaret sententiae, te ubique nostrum nominavi vix. Iusto paulo nonumy vim cu, eu summo urbanitas philosophia mel.</p>
+    <img src="http://www.nicoclub.com/articles/images/nt05.jpg"/>
+  </div>
+
```
Comment (fasteddiegarcia) [inline 02LessonTwo/in-class-project/index.html]: great work! dont forget to add in the contact information at the bottom.

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -3,8 +3,198 @@
...
+    </div>
+  </div>
+  <section class="hobbies">
+    <div class="container">
+      <div class="row">
+        <a name="hobbies"></a>
+        <div class="col-md-6">
+          <!-- couldn't figure out how to put a youtube video in without using iframe. was trying to do it with video or embed, but didn't have any luck -->
+          <video width="320" height="240" controls="no">
+            <source src="https://youtu.be/nneu8OC-0zM" type="video/mp4">
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html:107]: You may have needed to use the "Embedded" code here. Very nice try, though.

File: 02LessonTwo/homework/index.html [hunk 3]
```diff
@@ -3,8 +3,198 @@
...
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
 </head>
 <body>
+  <nav>
+      <!-- ask about Nav semantics and how it actually effects your content -->
+      <div class="row">
+        <div class="col-md-12">
+
+            <a href="#bio">Bio</a>
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html]: Well done.

File: 02LessonTwo/homework/index.html [hunk 4]
```diff
@@ -3,8 +3,198 @@
...
+            <a href="#hobbies">Hobbies</a>
+            <a href="#table">Table</a>
+            <a href="#resources">Resources</a>
+            <a href="#tv">TV Shows</a>
+
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html:25]: Well done with the anchored links in navigation.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 5]
```diff
@@ -5,6 +5,41 @@
...
+        <div class="footright">
+          <a href="https://www.facebook.com/james.puckett.35"><img src="https://cdn0.iconfinder.com/data/icons/yooicons_set01_socialbookmarks/256/social_facebook_box_blue.png" alt="Facebook"></a>
+          <a href="#"><img src="http://files.softicons.com/download/social-media-icons/clean-simple-social-icons-by-creative-nerds/png/256x256/Twitter.png" alt="Twitter"></a>
+          <a href="#"><img src="http://www.democrathotsprings.com/Instagram-icon-small.png" alt="Instagram"></a>
+        </div>
+      </footer>
     </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:47]: The only project requirement you are missing is the "Add Copyright and Year to footer". If you can add and recommit, I can bump your grade up to 100.

**Pull request closed without merging**